core/services/fluxmonitorv2/poll_manager.go

```diff
@@ -4,13 +4,17 @@ import (
 	"time"
 
 	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/flux_aggregator_wrapper"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+const hibernationPollPeriod = 24 * time.Hour
+
 type PollManagerConfig struct {
 	IsHibernating      bool
 	PollTickerInterval time.Duration
 	PollTickerDisabled bool
-	IdleTimerInterval  time.Duration
+	IdleTimerPeriod    time.Duration
```

Comment:
The main reason I have left it different is to indicate the difference between the `Ticker` and `Timer`, since the ticker ticks continuously after each interval, while a timer just ticks once at the end of the period.